tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+import os
+import pytest
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def pytest_addoption(parser):
+    parser.addoption("--lang", action="store", default="ts")
+    parser.addoption("--api", action="store", default="rest")
+    parser.addoption("--auth", action="store", default=False)
+    parser.addoption("--fs", action="store", default=False)
```

Comment:
Nit: could simplify so that we don't need to explicitly pass a value of "true". It's true if the flag is supplied, otherwise false

```suggestion
    parser.addoption("--auth", action="store_true")
    parser.addoption("--fs", action="store_true")
```